src/screens/menu/components/MenuButton.tsx

```diff
@@ -33,4 +35,8 @@ const styles = StyleSheet.create({
     justifyContent: 'space-between',
     alignItems: 'center',
   },
+  menuBoxWrap: {
+    minWidth: 200,
```

Comment:
Looks like this is the change that actually fixes the issue. I reverted all the other changes and introduced this one and that fixed it.